airbyte-webapp/src/services/workspaces/WorkspacesService.tsx

```diff
@@ -76,7 +76,7 @@ function useWorkspaceApiService() {
 }
 
 export const useCurrentWorkspaceId = () => {
-  const { params } = useRouter<unknown, { workspaceId: string }>();
+  const params = useParams() as { workspaceId: string };
```

Comment:
I changed current version to something like this 
```
export const useCurrentWorkspaceId = () => {
  const params = useParams<{ workspaceId: string }>();

  return params.workspaceId as string;
};
```

but still need to cast `workspaceId` to `string` since it is giving TS errors that value might be undefined. Is that okey,?